sass/components/_variables.scss

```diff
@@ -47,6 +47,7 @@ $link-color: color("light-blue", "darken-1") !default;
 // 2. Badges
 // ==========================================================================
 
+$badge-text-color: color('grey', 'darken-1') !default;
```

Comment:
Why there is a need to add **!default**?
I tried removing it, getting warning like  
`Unknown `grey` - `darken-1` in $colors.
         on line 368 of tmp/sass_compiler-input_base_path-DWtJXwMd.tmp/2/components/_color-variables.scss, in function `color``